models/articles.model.js

```diff
@@ -7,9 +8,24 @@ exports.fetchArticleById = (article_id) => {
       if (result.rows.length === 0) {
         return Promise.reject({ status: 404, msg: "Article doesn't exist!" });
       }
-      
-      return {article: result.rows[0]};
+
+      return { article: result.rows[0] };
     });
 };
 
+exports.fetchArticles = () => {
+  const query = `SELECT articles.author,articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url,
+    COUNT (comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC;`;
 
+  return db.query(query).then((result) => {
+    console.log(result.rows);
+    if (result.rows.length === 0) {
+      return Promise.reject({ status: 404 });
+    } 
```

Comment:
Would also be nice to send an error message to the user instead of just a status code